Services/CO.CDP.Organisation.WebApi/UseCase/UpdateOrganisationUseCase.cs

```diff
@@ -195,6 +194,20 @@ await organisationRepository.SaveAsync(organisation,
         return await Task.FromResult(true);
     }
 
+    private static async Task ValidateIdentifierIsNotKnownToUs(IOrganisationRepository organisationRepository, OrganisationIdentifier identifier)
+    {
+        if (string.IsNullOrWhiteSpace(identifier.Id))
```

Comment:
Are you sure about this? Just double-checking as I'm not sure right now if this use case is used to persist the "Other" identifier that has the `Id` set to `null`. Do we want to disallow "Other" identifiers here?

Reply:
there is a separate ticket in the backlog to update additionalIdentifiers so it will be dealt there.

Reply:
I'm not sure if I understand your response.

We didn't use to throw an exception if the identifier id is empty. Now we do. Did you make sure this change didn't break adding/updating an "Other" identifier that has the Id empty?

Reply:
user cant add 'Other' identifier on the update organisation in current scenario so it is fine for now. But when Update Identifier ticket will come into picture then It might require removing this check but I dont know use case for that.
If you see that this check is not required then I can remove it?

Reply:
No, leave it in. I just needed to make sure you checked your bases.